docs/operations.rst

```diff
@@ -62,6 +72,12 @@ Your output should resemble:
 
 For more information about Kafka Streams metrics, see :ref:`streams_monitoring`.
 
+===================================
+Starting and Stopping KSQL Clusters
```

Comment:
This should be more at the top, I'd suggest right after the first sub-section "Local Development and Testing with Confluent CLI".

Starting and stopping is the first thing you'll need to do (if you are not using Confluent CLI), and flow-wise it should also be before Health checks, Logging, etc.